lib/ansible/module_utils/network/eos/eos.py

```diff
@@ -599,9 +599,12 @@ def is_json(cmd):
 
 
 def is_local_eapi(module):
-    transport = module.params['transport']
-    provider_transport = (module.params['provider'] or {}).get('transport')
-    return 'eapi' in (transport, provider_transport)
+    transports = []
```

Comment:
I'm not sure about this. Would the module support multiple transports? If so, maybe it can be added as some kind of config.

Reply:
This isn't really for the module... this will always be `False` for resource modules, as the lack both `transport` and `provider`. The issue is that the eos module_utils routes everything through this function to handle local connections properly, so it makes some assumptions about the shape of modules.

We should probably avoid all of this when local is not expected, but that is a bigger issue than I wanted to tackle here.